db/migrate/20160115181255_add_manifest_tracking_number_column.rb

```diff
@@ -0,0 +1,22 @@
+class AddManifestTrackingNumberColumn < ActiveRecord::Migration
+  class Manifest < ActiveRecord::Base
+  end
+
+  def up
+    add_column :manifests, :tracking_number, :string
+
+    Manifest.all.each do |manifest|
+      tracking_number = "undefined-tracking-number-#{manifest.id}"
+      if manifest.content && manifest.content['generator'] && manifest.content['generator']['manifest_tracking_number']
+        tracking_number = manifest.content['generator']['manifest_tracking_number']
+      end
+      manifest.update_column(:tracking_number, tracking_number)
+    end
+
+    change_column :manifests, :tracking_number, :string, null: false, unique: true
```

Comment:
this migration will probably fail on prod since we have manifests that were creating before the manifest tracking nuber validation. how should we handle that?